File: cs/examples/plus.cs
using System;
using static NaesungMath.CS.Formulas;

class Program
{
    static void Main(string[] args)
    {
        int a = 1;
        int b = 2;

        var result = plus(a, b);

        Console.WriteLine($"{a} + {b} = {result}");
    }
}

File: cs/examples/plus.py
import sys
import os
sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'src')))

import naesungmath

print(f"NaesungMath v{naesungmath.VERSION}")

a = 1
b = 2

result = naesungmath.plus(a, b)

print(f"{a} + {b} = {result}")
